From f4dadb3850ef15dd7898d07fca72d2fa7e32ca6f Mon Sep 17 00:00:00 2001
From: Stephen Shkardoon <ss23@ss23.geek.nz>
Date: Mon, 25 Jun 2018 20:05:18 +1200
Subject: [PATCH] Use AC_COMPILE instead of AC_RUN to check for execinfo.h

This way enables cross compiling, since we don't need to run anything
during the configure script.
---
 scripts/common.m4 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/common.m4 b/scripts/common.m4
index ff023928..b6d051f5 100644
--- a/scripts/common.m4
+++ b/scripts/common.m4
@@ -153,7 +153,7 @@ dnl   Need to fix this so that it uses the stuff defined by the system.
 AC_DEFUN([TORRENT_CHECK_EXECINFO], [
   AC_MSG_CHECKING(for execinfo.h)
 
-  AC_RUN_IFELSE([AC_LANG_SOURCE([
+  AC_COMPILE_IFELSE([AC_LANG_SOURCE([
       #include <execinfo.h>
       int main() { backtrace((void**)0, 0); backtrace_symbols((char**)0, 0); return 0;}
       ])],
